src/arche/readers/items.py

```diff
@@ -29,8 +29,8 @@ def process_df(df: pd.DataFrame) -> pd.DataFrame:
         return df
 
     @staticmethod
-    def categorize(df: pd.DataFrame) -> pd.DataFrame:
-        """Cast columns with repeating values to `category` type to save memory"""
+    def categorize(df: pd.DataFrame) -> None:
+        """Cast columns with repeating values to `category` type inplace to save memory"""
```

Comment:
:lipstick: 

```suggestion
        """Cast columns with repeating values to `category` type in place to save memory"""
```

Reply:
Inplace is a valid word :)

Reply:
I cannot find it in any online English dictionary :(

Reply:
Google translate failed me, it didn't suggest the correction :)